db/migrate/20180312163100-create-score.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('scores', {
+      uri: {
```

Comment:
I believe postgresql's default collation is case sensitive. We may want to enable case insensitive for uri's, or downcase everything before storing/querying.